src/commands/pack/tarballs.ts

```diff
@@ -25,8 +24,8 @@ This can be used to create oclif CLIs that use the system node or that come prel
     const {flags} = await this.parse(PackTarballs)
     const targets = flags.targets.split(',')
     const buildConfig = await Tarballs.buildConfig(flags.root, {xz: flags.xz, targets: targets})
-    if (targets.includes('darwin-arm64') && !semver.gt(buildConfig.nodeVersion, '16.0.0')) {
-      throw new Error('darwin-arm64 is only supported for node >=16.0.0')
```

Comment:
This shouldn't be removed. If the user explicitly passes in `darwin-arm64` as a target (i.e. runs `oclif pack:tarballs --targets darwin-arm64`) but is trying to use node 14 or less this should still error. It's only in the case where they don't pass in any targets and they're using node 14 or less that this should just output a warning.

Should just put it above the `buildConfig` assignment instead.

Reply:
The targets flag defaults to all targets See https://github.com/oclif/oclif/pull/711/files#diff-8c957c644d989965c3dad37783dff849e57acb2933bf0e3f7c9bbb7680866996L16

Reply:
Ah, good catch. Let's just get rid of the default on the flag then since [`Tarballs.buildConfig()` already uses the defaults if nothing is passed in anyways](https://github.com/oclif/oclif/pull/711/files#diff-8a37a245e4dc9c5cd9977d42aec01025fa1a848c690db79a8dc60087ef607d64R70).

Reply:
@RodEsp I'm not sure I understand why we should get rid of the default? At the very least it allow us to communicate in the flag description that the default behavior is to do all targets

Reply:
@mdonnalley hmm... yes... you're right about conveying the default behavior via the flag help.

My apologies, my suggestions were a little precipitous but overall what I want to see is an error if the user explicitly passes in `darwin-arm64` as a target and is trying to use node 14 or less and just a warning about `darwin-arm64` if the user doesn't pass in any targets and is using node 14 or less, as Pete said.

Reply:
Another alternative would be to alter the default list based on node version. Then on node versions < 16 specifying arm would result in an invalid option error. This way we get errors thrown and target alignment by node version.

Reply:
@mdonnalley @RodEsp have we come to a consensus here so I might continuing with QA?

Reply:
@RodEsp Does this work for you? 
![Screen Shot 2021-11-16 at 1 17 03 PM](https://user-images.githubusercontent.com/10244328/142059145-35df3afc-0945-499d-92c2-76fd45f38343.png)

@peternhale help is cached at release time, so we're not able to alter the default based on the node version customers are using

Reply:
@mdonnalley yep that looks fine but will that also happen if the user runs `oclif pack:tarballs --targets darwin-arm64,darwin-x64`?

I'd prefer an error in that scenario as well, anytime the user explicitly passes in `darwin-arm64`.

Reply:
```
~/oclif-git/oclif/bin/run pack:tarballs --targets darwin-arm64,darwin-x64
 ›   Warning: darwin-arm64 is only supported for node >=16.0.0. Skipping...
```

Reply:
@RodEsp so you expect the behavior to be:
- warn and skip `darwin-arm64` if the user does not supply `--targets` flag and node is not 16+
- throw error if `darwin-arm64` is specifically supplied by the user and node is not 16+

What happens if the user supplies all the targets? Do we error or warn?

Personally, I think the existing approach is simpler and more consistent:
- warn and skip `darwin-arm64` if node is not 16+
- throw error if no valid targets are specified

Reply:
There is no easy way to determine if the user supplies `--targets` or it is using the defaults. May not be worth the effort. I support skipping arm with warning.

Reply:
Pete's right that there's no way to distinguish between the user supplying all the default targets and the user not supplying the flag at all

At this point, I think we should consider this good enough and wait for feedback from users